doc/environment.yml

```diff
@@ -25,8 +25,6 @@ dependencies:
     - pytest
     # for %%sql --interact
     - ipywidgets
-    # 0.13.2 is broken currently
     # https://github.com/pydata/pydata-sphinx-theme/issues/1274
-    - pydata-sphinx-theme<0.13.2
```

Comment:
this change doesn't look relevant, looks like this is related to the other PR about the pydata sphinx version

Reply:
move it to the other PR now